dask_cuda/benchmarks/local_cupy.py

```diff
@@ -263,11 +249,25 @@ async def run(args):
                 print_key_value(key="NVLink", value=f"{args.enable_nvlink}")
             print_key_value(key="Worker thread(s)", value=f"{args.threads_per_worker}")
             print_separator(separator="=")
-            print_key_value(key="Wall clock", value="npartitions")
+            print_key_value(key="Wall clock", value="Partitions")
             print_separator(separator="-")
-            for (took, npartitions) in took_list:
-                t = format_time(took)
-                print_key_value(key=f"{t}", value=f"{npartitions}")
+            times = []
+            for idx, (took, npartitions) in enumerate(took_list):
+                m = format_time(took)
+                times.append(took)
+                print_key_value(key=f"{m}", value=npartitions)
+            times = np.asarray(times)
+            bandwidths_all = np.asarray(bandwidths_all)
+            print_separator(separator="=")
+            print_key_value(
```

Comment:
I think we can deduce throughput for these operations as well, but it would need to be done for each op in turn, and we'd need to pick a cache model. For example does `(A + A.T).sum()` move `A.nbytes` of data or `2*A.nbytes` (or more)?

Reply:
The amount of data that gets moved (if by move you mean the same as transferring between workers) in the cuDF benchmarks is irrelevant for throughput, instead the throughput is based on the `data_processed` which is a sum of the DataFrames sizes:

https://github.com/rapidsai/dask-cuda/blob/a2e322362515bf612f9368eee99e150e6fa49437/dask_cuda/benchmarks/local_cudf_merge.py#L169-L170

In the same manner, I believe the equivalent `data_processed` for CuPy benchmarks here would be the sum of sizes of input arrays, e.g., `x.nbytes` for 

https://github.com/rapidsai/dask-cuda/blob/a2e322362515bf612f9368eee99e150e6fa49437/dask_cuda/benchmarks/local_cupy.py#L35

and `x.nbytes + y.nbytes` for

https://github.com/rapidsai/dask-cuda/blob/a2e322362515bf612f9368eee99e150e6fa49437/dask_cuda/benchmarks/local_cupy.py#L44-L45

IOW, the operation `A.T` size is not an input and thus doesn't get accounted for in `data_processed`. Does the above make sense?

Reply:
I meant data movement in the local processing, but I guess this is a slightly philosophical argument. I think I was thinking from the point of view of what is the streaming data cost? e.g. if I just compute `B = A.T` then I need to stream `A` and stream the output to `B` so the data movement is 2*A.nbytes (on cpus at least it would normally be 3*A.nbytes since caches are usually write-allocate without hinting that the output can be done with a non-temporal store). This kind of metric tells us something about the performance on the hardware.

But I agree that a throughput of "this is how much data I started with, how fast can I process it?" is also a good metric, and is generally the one we actually care about since that's the wallclock time question. 

In conclusion: happy to go with sum of input array sizes.